src/main/java/com/suse/saltstack/netapi/calls/modules/Match.java

```diff
@@ -0,0 +1,61 @@
+package com.suse.saltstack.netapi.calls.modules;
+
+import com.google.gson.reflect.TypeToken;
+import com.suse.saltstack.netapi.calls.LocalCall;
+import com.suse.saltstack.netapi.utils.ClientUtils;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * salt.modules.match
+ */
+public class Match {
+
+    public static LocalCall<Boolean> compound(String tgt, Optional<String> minionId) {
```

Comment:
Bindings that are not documented are crappy. It is not a port of Salt in Java that we could just refer to the original documentation. E.g. ZMQ has bindings [documented](http://zeromq.github.io/jzmq/javadocs/org/zeromq/ZMQ.Socket.html) although they do replicate the original docs.